torchbenchmark/models/maml_omniglot/__init__.py

```diff
@@ -0,0 +1,110 @@
+# This file was adapted from
+# https://github.com/facebookresearch/higher/blob/master/examples/maml-omniglot.py
+# It comes with the following license.
+#
+# Copyright (c) Facebook, Inc. and its affiliates.
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+import torch
+import torch.optim as optim
+import torch.nn as nn
+import torch.nn.functional as F
+from pathlib import Path
+import higher
+
+from ...util.model import BenchmarkModel
+
+
+class Model(BenchmarkModel):
+    def __init__(self, device=None, jit=False):
+        super().__init__()
```

Comment:
Please specify a task to the model: https://github.com/pytorch/benchmark/blob/master/torchbenchmark/tasks.py.

Reply:
This model is metalearning over classification on an image dataset. I think this is either OTHER.OTHER_TASKS or COMPUTER_VISION.OTHER_COMPUTER_VISION. Do you have advice for which category this should go @xuzhao9? (Otherwise I'll pick OTHER.OTHER_TASKS)

Reply:
> This model is metalearning over classification on an image dataset. I think this is either OTHER.OTHER_TASKS or COMPUTER_VISION.OTHER_COMPUTER_VISION. Do you have advice for which category this should go @xuzhao9? (Otherwise I'll pick OTHER.OTHER_TASKS)

Since it is meta learning, please add it to OTHER.OTHER_TASKS, thanks!

Reply:
Added OTHER.OTHER_TASKS.